contracts/FeralfileEnglishAuction.sol

```diff
@@ -0,0 +1,378 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+import "./IFeralfileSaleData.sol";
+import "./ECDSASigner.sol";
+import "./IFeralfileVault.sol";
+
+contract IFeralfileExhibitionV4 is IFeralfileSaleData {
+    function buyArtworks(
+        bytes32 r_,
+        bytes32 s_,
+        uint8 v_,
+        SaleData calldata saleData_
+    ) external payable {}
+}
+
+contract FeralfileEnglishAuction is Ownable, IFeralfileSaleData, ECDSASigner {
+    struct EnglishAuction {
```

Comment:
Minor. It could be `Auction` is enough since the contract is `FeralFileEnglishAuction` already.